packages/bot-web-ui/src/components/transactions/transactions.jsx

```diff
@@ -10,7 +10,7 @@ import Download from 'Components/download';
 import { contract_stages } from 'Constants/contract-stage';
 import { transaction_elements } from 'Constants/transactions';
 import { useDBotStore } from 'Stores/useDBotStore';
-import Transaction from './transaction.jsx';
+import Transaction from './transaction.tsx';
```

Comment:
For .tsx file we don't need to mention the extension @farabi-deriv 
```suggestion
import Transaction from './transaction';
```

Reply:
Right. Overlooked this. Thanks